README.md

```diff
@@ -22,18 +20,64 @@ Dedicated to making your workflow flow more seamless. Search everything from app
 
 <p align="center"> <sub>Remember to star it, flow will love you more :)</sub></p>
 
-<p align="center"><img width="12px" src="https://user-images.githubusercontent.com/26427004/104119722-9033c600-5385-11eb-9d57-4c376862fd36.png"> <a href="https://www.softpedia.com/get/System/Launchers-Shutdown-Tools/Flow-Launcher.shtml">SOFTPEDIA EDITOR'S PICK</a></p>
-
 <img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
 
 
-## 🎉 New Features in 1.9
+## 🎅 New Features🤶
+### Preview Panel
+<img src="https://user-images.githubusercontent.com/6903107/207159213-662999d3-2c18-4256-b473-c417efca0069.png" width="400">
+
+- Use the F1 key to open/hide the preview panel.
+- Media files will be displayed as large images, otherwise a large icon and entire path will be displayed.
+- Turn on preview permanently via Settings (Always Preview).
+- Use hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to adjust flow's search window width and height quickly if the preview area is too narrow.
+- This feature is currently in its early stages.
+
+### Everything plugin merged to Explorer
+<img src="https://user-images.githubusercontent.com/6903107/207171178-78912e76-31c2-4ba2-b3cf-de4b0b5e5691.png" width="300">
+
+- Everything's features are now included in the Explorer plug-in.
+- If you are already using everything plugin, you can only enable the explorer.
+- If you are using windows search indexer, add-ons are available from the Everything plug-in.
+
+### Clock & Dates in Search box
+
+<img src="https://user-images.githubusercontent.com/6903107/207159348-8b0c7a2b-0836-4764-916b-e0236087f7f3.png" width="400">
+
+- Display Time & dates in searchbox.
+
+### Drag and Drop
+<img src="https://user-images.githubusercontent.com/6903107/207159486-1993510f-09f2-4e33-bba7-4ca59ca1bc5a.png" width="500">
+
+- You can drag an item to a discord or desktop.
+- The drop target program determines whether to copy or move (You can change it CTRL or Alt), and is displayed in the mouse cursor.
+
+### Custom Shortcut
+
+<img src="https://user-images.githubusercontent.com/6903107/207170354-db1a98a6-3959-4341-82ed-0c417bdb121c.png" width="300">
+<img src="https://user-images.githubusercontent.com/6903107/207170525-3e99ad93-e8b7-4b39-b9b3-8fbeb443ed26.png" width="300">
+
+- Added shortcut functionality. This allows you to set additional action keywords.
```

Comment:
```suggestion
<img src="https://user-images.githubusercontent.com/6903107/207159213-662999d3-2c18-4256-b473-c417efca0069.png" width="400">

- Use the F1 key to open/hide the preview panel.
- Media files will be displayed as large images, otherwise a large icon and entire path will be displayed.
- Turn on preview permanently via Settings (Always Preview).
- Use hotkeys (Ctrl+Plus,Minus / Ctrl+],[) to adjust flow's search window width and height quickly if the preview area is too narrow.
- This feature is currently in its early stages.

### Everything Plugin Merged Into Explorer
<img src="https://user-images.githubusercontent.com/6903107/207171178-78912e76-31c2-4ba2-b3cf-de4b0b5e5691.png" width="300">

- Switch easily between Everything and Windows Search to take advantage of both search engines (remember to remove existing Everything plugin).
- Use features available to both Everything and Explorer plugins

### Date & Time Display In Search Window

<img src="https://user-images.githubusercontent.com/6903107/207159348-8b0c7a2b-0836-4764-916b-e0236087f7f3.png" width="400">

- Display the date and time when the search window is triggered.

### Drag & Drop
<img src="https://user-images.githubusercontent.com/6903107/207159486-1993510f-09f2-4e33-bba7-4ca59ca1bc5a.png" width="500">

- Drag an item to Discord or computer location.
- The target program determines whether the drop is to copy or move the item (can change via CTRL or Alt), and the operation is displayed on the mouse cursor.

### Custom Shortcut

<img src="https://user-images.githubusercontent.com/6903107/207170354-db1a98a6-3959-4341-82ed-0c417bdb121c.png" width="300">
<img src="https://user-images.githubusercontent.com/6903107/207170525-3e99ad93-e8b7-4b39-b9b3-8fbeb443ed26.png" width="300">

- New shortcut functionality to set additional action keywords or search terms.
```